eq-author/src/App/section/Design/SectionEditor/RepeatingSection/index.js

```diff
@@ -115,7 +116,7 @@ const RepeatingSection = ({ section, handleUpdate }) => {
         item from the list forms a section on the Hub. The repeating section is
         used to ask the same questions for each item selected.
       </Caption>
-      <InlineField>
+      <InlineField disabled={Boolean(!section.allowRepeatingSection)}>
```

Comment:
```suggestion
      <InlineField disabled={!section.allowRepeatingSection}>
```

`Boolean` can be removed as this value will be a boolean from the section data